R/log4j.properties

```diff
@@ -0,0 +1,28 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Set everything to be logged to the file target/unit-tests.log
+log4j.rootCategory=INFO, file
```

Comment:
This file was copied from the core Spark unit tests log4j.properties (https://github.com/apache/spark/blob/master/core/src/test/resources/log4j.properties)

The logging level is still INFO but its logged to a file named R-unit-tests.log. That way the console is not spammed but also you can see the INFO logs for debugging if required. 

We can probably put it inside R/pkg/inst/tests but I wasnt sure if this should be a part of the package.